From 54e0dc9f256010c2561f9c2f984989124f0230dd Mon Sep 17 00:00:00 2001
From: leeminju <leeminju@DESKTOP-93OGG00>
Date: Sun, 21 Oct 2018 23:27:20 +0900
Subject: [PATCH] .

---
 .../config/WebMvcContextConfiguration.java    | 60 +++----------------
 1 file changed, 7 insertions(+), 53 deletions(-)

diff --git a/vin/src/main/java/kr/ac/hongik/vin/config/WebMvcContextConfiguration.java b/vin/src/main/java/kr/ac/hongik/vin/config/WebMvcContextConfiguration.java
index a7e62b4..42c8536 100644
--- a/vin/src/main/java/kr/ac/hongik/vin/config/WebMvcContextConfiguration.java
+++ b/vin/src/main/java/kr/ac/hongik/vin/config/WebMvcContextConfiguration.java
@@ -1,51 +1,3 @@
-<<<<<<< HEAD
-package kr.ac.hongik.vin.config;
-
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Configuration;
-import org.springframework.web.servlet.config.annotation.DefaultServletHandlerConfigurer;
-import org.springframework.web.servlet.config.annotation.EnableWebMvc;
-import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
-import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
-import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
-import org.springframework.web.servlet.view.InternalResourceViewResolver;
-
-
-@Configuration
-@EnableWebMvc
-@ComponentScan(basePackages= {"kr.ac.hongik.vin.controller"})
-public class WebMvcContextConfiguration extends WebMvcConfigurerAdapter {
-	@Override
-    public void addResourceHandlers(ResourceHandlerRegistry registry) {
-        registry.addResourceHandler("/css/**").addResourceLocations("/resources/css/").setCachePeriod(31556926);
-        registry.addResourceHandler("/img/**").addResourceLocations("/resources/img/").setCachePeriod(31556926);
-        registry.addResourceHandler("/js/**").addResourceLocations("/resources/js/").setCachePeriod(31556926);
-        registry.addResourceHandler("/txt/**").addResourceLocations("/resources/txt/").setCachePeriod(31556926);
-    }
-	
-    // default servlet handler를 사용하게 합니다.
-    @Override
-    public void configureDefaultServletHandling(DefaultServletHandlerConfigurer configurer) {
-        configurer.enable();
-    }
-   
-    @Override	// default Controller를 설정.
-    public void addViewControllers(final ViewControllerRegistry registry) {
-        registry.addViewController("/").setViewName("index");
-    }
-    
-    @Bean	// 기본 경로는 src/main/webapp/WEB-INF/views 디렉토리이다.
-    public InternalResourceViewResolver getInternalResourceViewResolver() {
-        InternalResourceViewResolver resolver = new InternalResourceViewResolver();
-        //resolver.setPrefix("/WEB-INF/views/");
-        //resolver.setSuffix(".jsp");
-        resolver.setPrefix("frontend/");
-        resolver.setSuffix(".html");
-        return resolver;
-    } 
-}
-=======
 package kr.ac.hongik.vin.config;
 
 import org.springframework.context.annotation.Bean;
@@ -64,16 +16,16 @@ import org.springframework.web.servlet.view.InternalResourceViewResolver;
 @ComponentScan(basePackages= {"kr.ac.hongik.vin.controller"})
 public class WebMvcContextConfiguration extends WebMvcConfigurerAdapter {
 	@Override
-	// Resource들을 handling하는 핸들러를 선언 및 세팅
     public void addResourceHandlers(ResourceHandlerRegistry registry) {
         registry.addResourceHandler("/css/**").addResourceLocations("/resources/css/").setCachePeriod(31556926);
         registry.addResourceHandler("/img/**").addResourceLocations("/resources/img/").setCachePeriod(31556926);
         registry.addResourceHandler("/js/**").addResourceLocations("/resources/js/").setCachePeriod(31556926);
         registry.addResourceHandler("/txt/**").addResourceLocations("/resources/txt/").setCachePeriod(31556926);
+        registry.addResourceHandler("/frontend/**").addResourceLocations("/resources/static/").setCachePeriod(31556926);
     }
 	
     // default servlet handler를 사용하게 합니다.
-    @Override	// 아무런 핸들러가 정해지지 않았을 때, 사용하는 default servlet handler의 설정
+    @Override
     public void configureDefaultServletHandling(DefaultServletHandlerConfigurer configurer) {
         configurer.enable();
     }
@@ -86,9 +38,11 @@ public class WebMvcContextConfiguration extends WebMvcConfigurerAdapter {
     @Bean	// 기본 경로는 src/main/webapp/WEB-INF/views 디렉토리이다.
     public InternalResourceViewResolver getInternalResourceViewResolver() {
         InternalResourceViewResolver resolver = new InternalResourceViewResolver();
-        resolver.setPrefix("/WEB-INF/views/");
-        resolver.setSuffix(".jsp");
+        //resolver.setPrefix("/WEB-INF/views/");
+        //resolver.setSuffix(".jsp");
+        resolver.setPrefix("frontend/");
+        resolver.setSuffix(".html");
         return resolver;
     } 
 }
->>>>>>> c3bbc1c3034190fab947b57e3cd25e038328ec07
+
-- 
2.19.1.windows.1

